lms/djangoapps/instructor/views/api.py

```diff
@@ -1458,6 +1457,10 @@ def get_students_features(request, course_id, csv=False):  # pylint: disable=red
         query_features.append('team')
         query_features_names['team'] = _('Team')
 
+    if not is_integrity_signature_enabled(course_key):
+        query_features.append('verification_status')
+        query_features_names['verification_status'] = _('Verification Status')
+
```

Comment:
this will probably mess up someone because of the different order and bad parsing code

but I don't think it's worth the weird code to maintain the position of verification status and they can learn a valuable lesson about having their parsing code read the headers and not be position dependent

Reply:
"building character"

Reply:
@ashultz0 I updated the code so the ordering is maintained, let me know if this is a worse idea

Reply:
oh actually that's pretty clean, nice

no one has to learn anything unpleasant about parsing